src/containers.js

```diff
@@ -19,20 +19,25 @@ const overwriteComposeFile = async (filePath, fileContents) => {
   await fs.promises.writeFile(filePath, fileContents, 'utf-8');
 };
 
-const upgrade = async (fileName, fileContents) => {
+const update = async (fileName, fileContents, install = false) => {
   if (!fileName) {
     throw new Error('Invalid docker-compose file name');
   }
 
+  const filePath = path.join(dockerComposeFilePath, fileName);
+  if (install === fs.existsSync(filePath)) {
```

Comment:
Use `await fs.exists` instead. It's already an async method so it's not adding any complexity, and it's better for Node not to block on the filesystem.

Reply:
There is no async `fs.exists`. There's a long explanation about why [in the callback `exists` documentation](https://nodejs.org/dist/latest-v16.x/docs/api/fs.html#fsexistspath-callback). I can use `stat`, but it complicates the code somewhat.

Reply:
Oh interesting.  TIL `fs.exists` is deprecated, but `fs.existsSync` is not. That's so backwards. Let's roll with it for now.

Reply:
i already changed it to stat :)